about.html

```diff
@@ -0,0 +1,106 @@
+        </nav>
+    </header>
+    <section class="home-pages">
+        <div class="home">
+            <h1>CREATIVE COMMONS</br>GLOBAL SUMMIT 2015</h1>
+            <div class="hlsa"></div>
+        </div>
+        <div class="paragraphab">
+            <p>This is the Generic Global Summit, where we bring together the least influential and unimportant leaders from around the block to flap their gums about the non-issues and gossips of our time.A joyful celebration bellieving in the value of opennes and sharing, creating This is the Generic Global Summit, where we bring together the least influential and unimportant leaders from around the block to flap their gums about the non-issues and gossips of our time.A joyful celebration bellieving in the value<br> A posative change with people from over 80 countries is taking place in October, in Korea A posative change with people from over 80 countries is taking place in October, in Korea</p>
+        </div>
+        <div class="hom-foots">
+            <p>Please avoid doing anything. Here you can ask nothing</p>
+            <h4>rofl@gcgn.com</h4>
+        </div>
+    </section>
+    <section class="m-page">
+        <div class="primary">
+            <p>Creative Commons Global Summit 2015 Logo</p>
+            <div class="hlsa"></div>
+        </div>
+        <div class="secondary">
+            <p>This is the Generic Global Summit, where we bring together where we bring together where we bring the least influential and unimpThis is the Generic Global Summit, where we bring together the least influential and unimpThis is the Generic Global Summit, where we bring together the least influential and unimp</p>
+        </div>
+        <div class="third">
+            <img src="images/logo.png">
+        </div>
+    </section>
+    <hr class="just">
+    <section class="m-page">
+        <div class="primary">
+            <p>See the past CC Global Summits</p>
+            <div class="hlsa"></div>
+        </div>
+        <div class="secondary">
+            <p>This is the Generic Global Summit, where we bring together where we bring together where we bring the</p>
+        </div>
+        <div class="thirds">
+            <div class="thirdl">
+                <h1>2015</h1>
+                <p>CC Global Summits 2015 in Ambo</p>
+            </div>
+            <div class="thirdl">
+                <h1>2015</h1>
+                <p>CC Global Summits 2015 in Ambo</p>
+            </div>
+        </div>
+    </section>
+    <section class="partner">
+        <div class="partner-title">
+            <p>Partners</p>
+            <div class="hls"></div>
+        </div> 
+        <div class="names">
+            <div class="img1"><img src="images/partner-logo-1.svg"></div>
+            <div class="img1"><img src="images/partner-logo-2.svg"></div>
+            <div class="img1"><img src="images/partner-logo-3.svg"></div>
+            <div class="img1"><img src="images/partner-logo-4.svg"></div>
+        </div>
```

Comment:
- [ ] As suggested by the previous reviewer, I would kindly suggest you work on the `About page`, partner section. the way your partner companies are aligning does not look like what we have in the original design, Kindly check it out:

| Your design 👇  | Behance design 👇  |
| ------------- | ------------- |
| ![cor-foot](https://user-images.githubusercontent.com/78634841/225774442-4dd1b2d0-4a62-48bd-9ad2-00ea4a31758b.png) | ![image](https://user-images.githubusercontent.com/78634841/225774532-14ceb365-bd92-49ed-86cb-49b9c346587d.png) |